test/subproc-specs.js

```diff
@@ -209,9 +216,9 @@ describe('SubProcess', function () {
 
   describe('#stop', function () {
     it('should send the right signal to stop a proc', async function () {
-      return await new B(async (resolve, reject) => {
+      return await new Promise((resolve, reject) => {
         let subproc = new SubProcess('tail', ['-f', path.resolve(__filename)]);
-        await subproc.start();
+        subproc.start();
```

Comment:
I am not sure it is a good idea to remove await from here

Reply:
same above

Reply:
> The executor function can also be an async function . However, this is usually a mistake, for a few reasons: If an async executor function throws an error, the error will be lost and won't cause the newly-constructed Promise to reject. This could make it difficult to debug and handle some errors.

from eslint docs

Reply:
I refactored those two tests to use simple `async` / `await`. Event listeners are sync functions anyway and don't support async ones (it kind of defeats the point of event listening, which is already async)